examples/mean/tests_mixedlang/test_mean.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import print_function
+from __future__ import division
+
+import cocotb
+from cocotb.triggers import Timer, RisingEdge, ReadOnly
+from cocotb.result import TestFailure
+from cocotb.monitors import BusMonitor
+from cocotb.scoreboard import Scoreboard
+
+import random
+
+CLK_PERIOD_NS = 100
+
+
+class StreamBusMonitor(BusMonitor):
```

Comment:
Is this just a copy of the other file?

Reply:
Yes, this file was copied to get something going quickly. I'll clean this up.